spec/models/assignment_spec.rb

```diff
@@ -0,0 +1,34 @@
+require 'rails_helper'
+
+RSpec.describe Assignment, type: :model do
+  i18n_scope = 'activerecord.errors.models.assignment.attributes'
+  let(:off) { FactoryGirl.build_stubbed(:offspring) }
+  let(:shift1) { FactoryGirl.build_stubbed(:shift) }
```

Comment:
If you use build_stubbed, you are not creating it, it is possible that it fails as you are testing relationships